e2e/nomostest/clean.go

```diff
@@ -103,10 +101,6 @@ func Clean(nt *NT) error {
 	if err := uninstallConfigSync(nt); err != nil {
 		return err
 	}
-	// Delete the Kubevirt resources
-	if err := deleteKubevirt(nt); err != nil {
```

Comment:
Still need to keep this to clean up clusters that errored during a previous run with the kubevirt applied.

Reply:
Add a TODO to delete the cleanup in the future, once all the test clusters are clean.

Reply:
Good point. Done.